SalesforceSDK/Universal/Analytics/AILTNPublisher.cs

```diff
@@ -85,7 +88,7 @@ public class AiltnPublisher : IAnalyticsPublisher
             }
             var path = string.Format(API_PATH, ApiVersionStrings.VersionNumber);
             var headers = new Dictionary<string, string>();
-            var request = new RestRequest(HttpMethod.Post, path, body.ToString(), ContentTypeValues.Gzip, headers);
+            var request = new RestRequest(HttpMethod.Post, path, body.ToString().Replace("\r", "").Replace("\n", "").Replace(" ", ""), ContentTypeValues.Json, headers);
```

Comment:
Same question as above. There has to be a better way in `Windows` for a `POST body`. This seems hacky.